CustomEntitySearchTest/WebAPISkillContract.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+
```

Comment:
don't copy these files here, just add a reference to them.
Same for WebAPISkillHelper.cs